Chap_API_Sync_Access.tex

```diff
@@ -157,8 +157,8 @@ \subsection{Fence-related attributes}
 has been committed via \refapi{PMIx_Commit}, making the collection locally
 available to each participant at the end of the operation. By default, this will include all job-level information that was locally generated by \ac{PMIx} servers unless excluded using the \refattr{PMIX_COLLECT_GENERATED_JOB_INFO} attribute.
 }
-%
-\declareAttributeProvisional{PMIX_LOCAL_COLLECTIVE_STATUS}{"pmix.loc.col.st"}{pmix_status_t}{
+
+\declareAttributeNEW{PMIX_LOCAL_COLLECTIVE_STATUS}{"pmix.loc.col.st"}{pmix_status_t}{
```

Comment:
```suggestion
\declareAttributeProvisional{PMIX_LOCAL_COLLECTIVE_STATUS}{"pmix.loc.col.st"}{pmix_status_t}{
```
To preserve the change in https://github.com/pmix/pmix-standard/commit/77a64f3e22280c2199fb0450e931496d1a4d1045#diff-d2563d457cff789a47a207bed39220b1a8520e6165aaf48a957d5ecd0953849b

Reply:
Fixed during the rebase.